### DISCUSSION TITLE
Why is dist` folder not purged on rebuild?

### DISCUSSION THREAD

**Discussion by avalanche1** [parcel-bundler/parcel#5869] [Q&A] [answered]
With time `dist` folder gets full with garbage, e.i. obsolete files from prev builds.
I'm devving a chrome extn and oftentimes have to consult this folder to see what is actually in the dist files fed to the browser. All those remnants of the good ol' days - they really obscure the eye sometimes.

2nd q: Is there a setting to force-purge it on rebuild (not only rebuild, but `parcel serve` as well)

**Comment by mischnic** [comment, marked as answer]
1. With parcel build, you could do `rm -rf dist` before running `parcel build`
2. See https://github.com/parcel-bundler/parcel/issues/3591
